lib/logstash/environment.rb

```diff
@@ -33,6 +35,18 @@ def logstash_gem_home
       ::File.join(BUNDLE_DIR, ruby_engine, gem_ruby_version)
     end
 
+    def env
+      LOGSTASH_ENV
+    end
+
+    def method_missing(method_name, *args)
+      if (method_name[-1] == "?")
+        env == method_name[0..-2]
+      else
+        raise NoMethodError.new("undefined method #{method_name} for LogStash::Environment");
+      end
+    end
```

Comment:
personally I don't really like this method_missing magic for this context, it is really hard to understand what the API is. I'd prefer to have explicit methods.